_data/simple-icons.json

```diff
@@ -11171,6 +11171,11 @@
             "source": "https://brand.unity.com/",
             "guidelines": "https://brand.unity.com/"
         },
+        {
+            "title": "UnoCSS",
+            "hex": "000000",
```

Comment:
The icon uses `#333333` as a fill color (the shapes have varying opacity in the original icon though). I think there's no clear brand color for this project, but the fill color might be a better choice IMO. @sachinraja What do you think?

Reply:
Yes let's go with `#333333` :+1: